recipes/t_coffee/t_coffee/config.py

```diff
@@ -1,27 +1,27 @@
 import os
 import os.path as op
+import tempfile
 
 tcoffee_install_dir = op.normpath(op.join(op.dirname(__file__), '..', '..', '..', '{{TCOFFEE_FOLDER_NAME}}'))
 tcoffee_bin_dir = op.join(tcoffee_install_dir, 'bin')
 tcoffee_exe_file = op.join(tcoffee_bin_dir, 't_coffee')
 tcoffee_plugins_dir = op.join(tcoffee_install_dir, 'plugins', 'linux')
 tcoffee_perl_dir = op.join(tcoffee_install_dir, 'perl', 'lib', 'perl5')
-tcoffee_default_email = 'username@hostname.com'
+tcoffee_default_email = 'username@example.org'
+
 
 def get_tcoffee_environ():
     env = os.environ.copy()
-    if 'DIR_4_TCOFFEE' not in env:
-        env['DIR_4_TCOFFEE'] = tcoffee_install_dir
-    elif tcoffee_install_dir not in env['DIR_4_TCOFFEE']:
-        env['DIR_4_TCOFFEE'] = env['DIR_4_TCOFFEE'] + ':' + tcoffee_install_dir
```

Comment:
It seems to be used a lot to determine the location of other directories, see https://github.com/cbcrg/tcoffee/blob/master/lib/util_lib/util.c

Reply:
I think `TMP_4_TCOFFEE` was set so that multiple TCoffee instances can run concurrently without crashing each other. It may make sense to keep that setting.

Reply:
So we should set it isn't it?

Reply:
I think for a "normal" use case having `.t_coffee/cache`, `.t_coffee/plugins` inside the user home directory is fine (surely better than inside the conda env). For the Galaxy wrapper, we can set HOME inside the job working directory with the new `use_shared_home="false"` attribute.